### PR TITLE
First Draft

### PR DESCRIPTION
**Pull request by DanCh11**
word2vec with SVM model, using "Type of Measure, Category and Target groups" labels

### CODE REVIEW

File: doc/todo [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# de pus in array mare, dupa care in set,
```
Comment (costezki) [inline doc/todo:1]: we can delete this file.
@cosletmihai  can recommend a better way of handling the todo lists

File: src/evaluation_metrics.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+from sklearn.metrics import (confusion_matrix, f1_score, precision_score, accuracy_score,
+                             recall_score, mean_squared_error, mean_absolute_error)
+
+import pandas as pd
+
+class EvaluationMetrics:
```
Comment (costezki) [inline src/evaluation_metrics.py:6]: this can be a function, no need for the class

**Review by costezki — APPROVED** [review]
good start

File: doc/todo (hunk 1 above)
Comment (cosletmihai) [reply doc/todo:1]: community best practices can be found here: https://www.python.org/dev/peps/pep-0350/

but I usually just add a 
```python
# TODO: some stuff I need to do
```
`pycharm` supports this as well https://www.jetbrains.com/help/pycharm/using-todo.html

File: src/pwdb_transformer.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import jq
+import pandas as pd
+
+
+def transformer(file, rules):
```
Comment (costezki) [inline src/pwdb_transformer.py:5]: the function names should always reflect an action. 
Please refer to `function naming` chapter in the `clean code` book to get a feeling about what are good function names.

**Review by costezki — COMMENTED** [review]
some refactoring necessary
